src/Migrations/M021MigrateDefaultPostType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use P4\MasterTheme\Settings\DefaultPostType;
+
+class M021MigrateDefaultPostType extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    public static function execute(MigrationRecord $record): void
+    {
+        $value = get_option('default_p4_pagetype');
+
+        $options = get_option('planet4_options');
+        unset($options['default_p4_pagetype']);
```

Comment:
Hi @mleray , The old default p4 page type value is saved in `planet4_options` (i.e ``$options['default_p4_pagetype']), we are reseting old value that look fine, but on line # 29  we have to update the same value in new key, right? 🤔  (`update_option(DefaultPostType::KEY, $options['default_p4_pagetype']);`)

Reply:
Yeah, it's saved in the `$value` variable from line 19, no?

Reply:
By the time we get to line 29 `$options['default_p4_pagetype'])` is now unset so we can't use that

Reply:
Yes, we need to move line 29 before line 22 or save `default_p4_pagetype` into other var.

The `get_options()` function search the values for `option name`. 
Here in our case `get_option('default_p4_pagetype')` & `get_option('planet4_options')` return different values based on the option name we pass as a argument.
https://developer.wordpress.org/reference/functions/get_option/

Reply:
Ah okay, from a quick test on local it seems that on line 19 I should use `planet4_get_option` instead of `get_option`, is that what you mean?

Reply:
Similar to what we do in this [migration](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Migrations/M019MigrateReadingTime.php)

Reply:
That should work, the `planet4_get_option()` is a wrapper function on `get_option()` wp function. 🙂 
```
function planet4_get_option(string $key = '', $default = null)
{
    $options = get_option('planet4_options');

    return $options[ $key ] ?? $default;
}
```

Reply:
Updated!